generative_ai/gemini_tuning.py

```diff
@@ -59,7 +65,7 @@ def gemini_tuning_advanced(project_id: str) -> sft.SupervisedTuningJob:
     # TODO(developer): Update and un-comment below lines
     # project_id = "PROJECT_ID"
```

Comment:
Should we consider change the comment? project_id is not valid anymore.

Reply:
I agree